specification/translation/data-plane/Azure.AI.DocumentTranslation/readme.md

```diff
@@ -7,20 +7,20 @@ Configuration for generating Document Translation  SDK.
 ### Basic Information
 
 These are the global settings for the app.
-The current release is `2024-05-01`.
+The current release is `release-2024-05-01`.
```

Comment:
```suggestion
The current release is `package-2024-05-01`.
```

The standard is to use name `package-YYYY-MM-DD` rather than `release-YYYY-MM-DD`.

Reply:
Updated